libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -765,6 +765,11 @@ auto WaveTrack::CopyOne(
    const auto &pFactory = track.mpFactory;
    auto result = track.EmptyCopy();
    WaveTrack *newTrack = result.get();
+   const auto& tempo = track.GetProjectTempo();
+   // Even if `forClipboard`, we need some project tempo to place the clips
+   // correctly. When pasting, the track's tempo shall be set to the destination
+   // project tempo.
+   newTrack->mProjectTempo = tempo;
```

Comment:
I think this line is unnecessary because `Track::Init` was called in `EmptyCopy` and that copied the tempo.